app.py

```diff
@@ -601,14 +602,22 @@ def reaper_endpoint():
     """
     Endpoint that just runs the reaper once and returns the status
     """
-    logger.info({"message": "Reaper endpoint called"})
-    try:
-        num = reaper()
-        resp = flask.Response("Reaper success: {} deleted".format(num))
-        resp.status_code=200
-    except Exception as ex:
-        resp = flask.Response("Reaper error: {}".format(repr(ex)))
-        resp.status_code=500
+
+    request = flask.request
+    logger.info({"message": "Reaper endpoint called from {}".format(request.remote_addr)})
+
+    resp = flask.Response("generic Reaper response")
+    if (cfg['reaper_allowed_ip'] == request.remote_addr or cfg['reaper_allowed_ip'] == "0.0.0.0"):
```

Comment:
Maybe change it to a prefix match using startswith()? That way we could restrict it to a subnet if we wanted to?

Reply:
If we want to do that we should just do real CIDR matching instead.

Reply:
Added ipaddress module to support one subnet.